app/routes.js

```diff
@@ -2,15 +2,43 @@ var express = require('express');
 var router = express.Router();
 
 router.get('/', function (req, res) {
+  
   res.render('index');
+
 });
 
 
-// Example route: Passing data into a page
+// Example routes - feel free to delete these
+
+// Passing data into a page
+
 router.get('/examples/template-data', function (req, res) {
+
   res.render('examples/template-data', { 'name' : 'Foo' });
+
+});
+
+// Branching
+
+router.get('/examples/over18', function (req, res) {
+
+  // get the answer from the query string (eg. ?over18=false)
+	var over18 = req.query.over18;
+
```

Comment:
Would it be worth it to have a default? Or make it clearer that if there is no query string, x will happen?